tests/system/action/user/test_save_saml_account.py

```diff
@@ -448,3 +448,185 @@ def test_create_saml_account_boolean_integer_types_error(self) -> None:
         )
         self.assert_status_code(response, 400)
         self.assertIn("Could not parse 2, expect boolean", response.json["message"])
+
+
+class UserAddToGroup(UserBaseSamlAccount):
+    def setUp(self) -> None:
+        super().setUp()
+        self.set_models(
```

Comment:
Move this to the other `set_models` call

Reply:
done